src/Controllers/DeleteEntryController.php

```diff
@@ -21,9 +21,33 @@ public function __construct($BlogModel)
 
     public function __invoke(Request $request, Response $response, array $args)
     {
-        $DeletedEntry = $request->getParsedBody();
-        $DbResult = $this->BlogModel->DeleteEntry($DeletedEntry);
-        return $response->withHeader('Location', '/');
+        $ResponseData =
+            [
+                'success' => false,
+                'message' => '',
+                'data' => []
+            ];
+
+        $BlogPost = new Valitron\Validator($_POST);
+        $BlogPost->rule('required', ['GUID']);
+
+        if ($BlogPost->validate()) {
+            $Result = $this->BlogModel->DeleteEntry($DeletedEntry);
+            if ($Result) {
+                $ResponseData['success'] = true;
+                $ResponseData['message'] = "Your post has been successfully deleted!";
+                $ResponseData['data'] = $Result;

```

Comment:
this will also simply return true I assume if the blogPost was successfully marked as deleted. seems a little pointless imo